docs/data/stashcache/install-cache.md

```diff
@@ -5,100 +5,130 @@ This document describes how to install a StashCache cache service.  This service
+
+- `ANY_PUBLIC` indicates that the cache is willing to serve public data from any VO.
+- `ANY` indicates that the cache is willing to serve data from any VO,
+  both public and non-public.
+  `ANY` implies `ANY_PUBLIC`.
+
+There are extra requirements for serving non-public data:
+
+- In addition to the cache allowing a VO in the `AllowedVOs` list,
+  that VO must also allow the cache in its `AllowedCaches` list.
+  See the page on [getting your VO's data into StashCache](/data/stashcache/vo-data).
+- There must be an authenticated XRootD instance on the cache server.
+- There must be a `DN` attribute in the resource
+  with the DN of the cert used by the authenticated XRootD instance.
+
+This is an example of a cache server that serves all public data:
+```yaml
+  MY_STASHCACHE_CACHE:
+    FQDN: my-cache.example.net
+    Service: XRootD cache server
+      Description: StashCache cache server
+    AllowedVOs:
+      - ANY_PUBLIC
+```
+
+This is an example of a cache server that only serves authenticated data from the OSG VO:
+```yaml
+  MY_AUTH_STASHCACHE_CACHE:
+    FQDN: my-auth-cache.example.net
+    Service: XRootD cache server
+      Description: StashCache cache server
+    AllowedVOs:
+      - OSG
+    DN: /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=my-auth-cache.example.net
+```
+
+
+### Finalizing registration
+
+Once initial registration is complete, you may start the installation process.
+In the meantime, open a [help ticket](https://support.opensciencegrid.org) with your cache name.
+Mention in your ticket that you would like to "Finalize the cache registration."
+
+
+Installing the Cache
+--------------------
+
+The StashCache software consists of an XRootD server with special configuration and supporting services.
+To simplify installation, OSG provides convenience RPMs that install all required
+packages with a single command:
+
+    :::console
+    root@host # yum install --enablerepo=osg-testing stash-cache
 
 
 Configuring the Cache
 ---------------------
 
 First, you must create a "cache directory", which will be used to store downloaded files.
-By default this is `/stash`.
+By default this is `/mnt/stash`.
 We recommend using a separate file system for the cache directory,
 with at least 1 TB of storage available.
 
 !!! note
     The cache directory must be writable by the `xrootd:xrootd` user and group.
 
-The `stashcache-cache-server` provides a default configuration file,
-`/etc/xrootd/xrootd-stashcache-cache-server.cfg`, which must be customized for your cache.
-
-The most common lines to customize are:
+The `stash-cache` package provides default configuration files in `/etc/xrootd/xrootd-stash-cache.cfg` and `/etc/xrootd/config.d/`.
+Administrators may provide additional configuration by placing files in `/etc/xrootd/config.d/1*.cfg` (for files that need to be processed BEFORE the OSG configuration) or `/etc/xrootd/config.d/9*.cfg` (for files that need to be processed AFTER the OSG configuration).
```

Comment:
Determining which of these categories your configuration falls into is left as an exercise for the user :P

Reply:
"To determine which setting falls into which category, ask #xcache on Slack"